aspnetcore/blazor/handle-errors.md

```diff
@@ -0,0 +1,214 @@
+* The current set of event-handling delegates that could be triggered by client-side events.
+
+If a user opens the app in multiple browser tabs, they have multiple independent circuits.
+
+Blazor treats most unhandled exceptions as fatal to the circuit where they occur. If a circuit is terminated due to an unhandled exception, the user can only continue to interact with the app by reloading the page to create a new circuit. Circuits outside of the one that's terminated, which are circuits for other users or other browser tabs, aren't affected. This scenario is similar to a desktop app that crashes&mdash;the crashed app must be restarted, but other apps aren't affected.
+
+The reason for terminating a circuit for most unhandled exceptions is that an unhandled exception often leaves the circuit in an undefined state. If the framework continues processing a component's code after an unhandled exception and regardless of the circuit's state, the app's normal operation can't be guaranteed. Even security vulnerabilities in the app may result.
+
+## Manage unhandled exceptions in developer code
+
+If you want users to be able to continue using an app after an error occurs (for example, when loading or saving data), develop the app with error handling logic. C# [try-catch](/dotnet/csharp/language-reference/keywords/try-catch) statements are useful in error handling scenarios and are a suggested approach in later sections of this article that describe potential sources of unhandled exceptions.
+
+The structure of a `try-catch` statement includes two *blocks* of code:
+
+* The `try` block contains code that may throw an exception.
+* The `catch` block contains code that executes when an exception occurs in the `try` block.
+
+```csharp
+try
+{
+    ...
+}
+catch (Exception ex)
+{
+    ...
+}
+```
+
+In many C# development scenarios, trapping *all exceptions* with <xref:System.Exception?displayProperty=fullName> isn't considered a *best practice*. Usually, a more specific exception is trapped because:
+
+* You should only catch those exceptions that you know how to recover from.
+* Specific exceptions can be handled in a fine-grained manner:
+  * Specific method calls and property settings can be made by the app for a specific error condition.
+  * Specific error messages can be logged and shown to the user.
+
+In Blazor exception scenarios, however, we recommend trapping all exceptions in components where unhandled exceptions may occur. Trapping all exceptions provides an opportunity to:
+
+* Update the rendered component UI with a general exception message for the user.
+* Log a general exception for debugging.
+
+This approach usually avoids a circuit failure and allows a user to continue interacting with a component.
+
+`try-catch` statements can be nested inside of other `try-catch` statements. When nested statements are used, more specific exceptions can be caught before the most general type of exception is trapped with `System.Exception`.
```

Comment:
Again, super weird to talk about this here

Reply:
For newbies ... but *sure thing*. I'm not super-excited about this coverage.